scripts/find_samples_needing_sv.py

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/python3
+
+"""
+a script to find samples that need to be run through the structural variant pipeline
+(i.e. are not present yet in any extant callsets)
+
+usage:
+    python3 find_samples_needing_sv.py
+    -n <min_#_samples>
```

Comment:
Can you add something that explains what -n does?

Reply:
updated! Also switched this to max_samples to reflect what it intends to do